Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -606,10 +609,13 @@
                         },
                         excelTemplate: dataItem => {
                             var reference = dataItem.Reference;
-                            if (reference != null) {
-                                return reference.Title;
+                            if (reference == null) {
+                                return "";
                             }
-                            return "";
+                            if (Utility.Validation.validateUrl(reference.ExternalReferenceId)) {
```

Comment:
Jeg ser for mig at disse linjer vil gentage sig mange steder.. .skal vi ikke lave en export helper der kan transformere en inputreference til en string og så kalder vi bare den alle steder.. den tjekker på null, validitet og outputter det rigtige i konteksten af excel export... Kunne f.eks. være 
`ExcelExportHelper.renderReference(dataItem.Reference)`